account_payment_return/README.rst

```diff
@@ -0,0 +1,32 @@
+Return payment orders
+=====================
+
+Features:
+---------
+
+ * This module implements the return of invoice payments.
```

Comment:
Explain a little more that allows to catch several reconciled account move lines, to give back to a state where the debt is still open, and letting history of it.